compose/ui/ui-text/src/desktopTest/kotlin/androidx/compose/ui/text/DesktopParagraphTest.kt

```diff
@@ -170,7 +490,7 @@ class DesktopParagraphTest {
                 width = width
             )
 
-            for (i in 0..ltrText.length) {
+            for (i in 0..ltrText.lastIndex) {
```

Comment:
```suggestion
            for (i in ltrText.indices) {
```
Also, you may use indices (it's the same)

Reply:
Thank you! done